packages/@atjson/source-html/src/converter/third-party-embeds.ts

```diff
@@ -214,6 +215,22 @@ export default function convertThirdPartyEmbeds(doc: Document) {
         })
       );
     });
+  /**
+   * CNE Ticketing Widget
+   */
+  doc.where({ type: "-html-cne-ticketing-widget" }).update((embed) => {
+    doc.replaceAnnotation(
+      embed,
+      new CneTicketingWidgetEmbed({
+        id: embed.id,
+        start: embed.start,
+        end: embed.end,
+        attributes: {
+          url: embed.attributes.url,
+        },
+      })
+    );
+  });
```

Comment:
Just a note - the purpose for this is only to support a conversion from this kind of contrived html snippet. If in the future users would have a different way of producing this kind of embed (for example, perhaps in a rich-text environment rather than inserting via an embed code, the user would have a form to enter in the loggedIn/loggedOut URLs and privacy flag) then we could remove this